### ISSUE TITLE
Deprecation msg on the use of sendAction thrown in Ember 3.4.x

### ISSUE THREAD

**Issue by HerveSeger** [kennethkalmer/ember-cli-nouislider#73]
Hi.

Thank you for this very useful addon.

I get the following message in the console with ember-cli-nouislider `v1.0.0`:

```
You called <insurance-online-ui@component:range-slider::ember574>.sendAction("on-update") but Component#sendAction is deprecated. Please use closure actions instead. [deprecation id: ember-component.send-action] See https://emberjs.com/deprecations/v3.x#toc_ember-component-send-action for more details.
```

Thanks.

**Comment by puwelous** [comment]
Should be solved in [release v1.1.0](https://github.com/kennethkalmer/ember-cli-nouislider/releases/tag/v1.1.0) via https://github.com/kennethkalmer/ember-cli-nouislider/pull/74

**Comment by HerveSeger** [comment]
Great! I will update then. 

Thanks a lot.
